examples/getting_started/3_Deep_dive_into_the_anatomy_of_quantum_circuits.ipynb

```diff
@@ -41,7 +41,6 @@
     "# magic word for producing visualizations in notebook\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    from braket.circuits import Circuit, Gate, Instruction, circuit, Observable
</pre><p>Is the "circuit" import necessary? (The lower case one, not the upper case one)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
It is used: `@circuit.subroutine`